rtl8720dn/wifi.go

```diff
@@ -0,0 +1,72 @@
+package rtl8720dn
+
+import (
+	"fmt"
+	"time"
+)
+
+func (r *RTL8720DN) ConnectToAP(ssid string, password string) error {
```

Comment:
In a future iteration we should create a interface for `wificlient` and another for `wifiaccesspoint` to have a uniform way to switch between this and wifinina and espat.

Reply:
I have the same opinion.